web/src/i18n/locales/en/apd/apd.yaml

```diff
@@ -1,35 +1,30 @@
 title: Program Summary
 helpText: >-
-  Provide an introduction for the state program. Include context for this
-  funding request by providing information such as current size of program,
+  Provide an introduction to the state program. Include context for this
+  funding request by providing information such as current size,
   future vision, and how the program is positioned within the state. This
   information is similar to the content from the IAPD executive summary
   template section.
 overview:
   title: Program Overview
-  helpText: >-
-    Provide a high level summary of the state program including history,
-    and future vision. Include history and summary of future vision of the
-    state program.
-  reminder: >-
-
   yearsCovered: Choose the federal fiscal year(s) this APD covers.
 
 introduction:
   title: Program Introduction
   helpText: >-
-    Provide an introduction for the program including context, history, and the state's future vision.
+    Provide a high-level introduction to the state program. Include the history and context of
+    the program and the state’s future vision for it.
 
 hit:
   title: HIT Overview
   helpText: >-
-    Briefly summarize planned HIT funded activities contained within this APD.
-  reminder: >-
+    Briefly summarize planned HIT-funded activities contained within this APD.
 
 hie:
   title: HIE Overview
   helpText: >-
-    Summarize planned HIE funded activities within this APD.
+    Summarize planned HIE-funded activities within this APD. If
+    there are no HIE activities, skip this section. Please include:
```

Comment:
Is it better to ask the user to enter "No HIE activities" or something like that vs just skipping the section?

Reply:
This is a programatic question, I think. A couple different questions to consider:

- Is it useful for analysts to have that stated affirmatively? (If so, we should ask them to say it.)
- Is it common for people to skip this when they actually should have said something? I'm thinking more often than an edge case, but wouldn't necessarily be most of the time. (If so, asking them to type something can help make sure they don't skip it.)

Reply:
Per your comment at the end of the PR, I'm going to backlog this question and not treat it as a show-stopper for now.

Reply:
@jameshupp In the APD's template, HIE content is focused in Appendix D (HIE) which most states would make a note that they do not have any HIE requests. Since we've mainstreamed that content, we could mirror that behavior here.

Also, thinking in the context of reading the APD, how would the analyst know that the state either 1) decided not to fill it out or 2)  accidentally skipped it?

Reply:
Love this line of thinking. Okay, let's do that. I'll add it in.